libr/io/p/io_r2k.c

```diff
@@ -65,7 +65,7 @@ static int r2k__close(RIODesc *fd) {
 	}
 #elif __linux__
 	if (fd) {
-		close (fd->fd);
+		close (fd->user);
```

Comment:
I will ask condret about this, because as per this issue (https://github.com/radare/radare2/issues/6634), he said to use user from RIODesc.